recipe/meta.yaml

```diff
@@ -27,8 +27,11 @@ requirements:
     - python
     - numpy
     - scipy
+    - setuptools
```

Comment:
I added `setuptools` to `run` requirement, because the latest Treelite needs access to `pkg_resources`:
https://github.com/dmlc/treelite/blob/76d8e79fa8082be867e7cc44c6c3096bec8d3309/python/treelite/frontend.py#L10